### DISCUSSION TITLE
Windows ARM Support

### DISCUSSION THREAD

**Discussion by etmartinkazoo** [beyondcode/herd-community#850] [General]
Is Laravel Herd for Windows updated to work natively with the new ARM processors? Thank you for the great software!

**Comment by jeffwhansen** [comment]
This would be a great addition.

**Comment by sXe79** [comment]
Waiting for ARM native app

**Comment by Sennik** [comment]
Me too!

**Comment by riggerz29** [comment]
Me too, I would love a native ARM for both Herd and Tinkerwell

**Comment by illu-teun** [comment]
Yes please!

**Comment by hdesoto** [comment]
Please, please!
